client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -148,8 +150,9 @@ public void update(ReduceFileGroups fileGroups) {
   protected final Map<Integer, ReduceFileGroups> reduceFileGroupsMap =
       JavaUtils.newConcurrentHashMap();
 
-  public ShuffleClientImpl(CelebornConf conf, UserIdentifier userIdentifier) {
+  public ShuffleClientImpl(String appUniqueId, CelebornConf conf, UserIdentifier userIdentifier) {
     super();
+    this.appUniqueId = appUniqueId;
```

Comment:
appuniqueId is removed in some print log. is it necessary to print here ?so that we can use this appuniqueId to look up  worker/master  log

Reply:
done